configs/11.0/packages/libpfm/sources

```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="Libpfm"
-ATSRC_PACKAGE_VER=4.7.1
-ATSRC_PACKAGE_REV=ebb8be7
+ATSRC_PACKAGE_VER=4.8.0
+ATSRC_PACKAGE_REV=f3c2293
```

Comment:
I guess it should be 13 characters. I did not change it because it was using 7 so I left that way. But I will change it to 13.